### DISCUSSION TITLE
will the mod be updated to 1.21?

### DISCUSSION THREAD

**Discussion by Carmabound** [LeahPy/ImmersiveThunder#10] [General]
all in the title

**Comment by netcatgirl** [comment]
Heya! I'll update it next week. Sorry for the delay :s

**Reply by Carmabound** [reply]
Thank you so much and no rush!!!!! I saw it in a mod compilation video and am really excited abt it :3

**Reply by Carmabound** [reply]
> Heya! I'll update it next week. Sorry for the delay :s

I actually just finished updating it to 1.21 (it works) because I wasn't sure if you would respond, but this is my first time really dealing with the inner workings of mods so I'm worried I might have left an absolute path in there that shows my name. If you want to check it, though, I can email it to you?
